### ISSUE TITLE
Docs Enhancements 

### ISSUE THREAD

**Issue by AmrTamer23** [childrentime/reactuse#77]
Thanks for this useful lib, but the docs need some enhancements like getting out of the docusaurus template and maybe make it docs only w/o the landing page, and I'm ready to help! Let me know if there is a room for this enhancement and I will work on it.

**Comment by childrentime** [comment]
Thank you for your willingness to help and for your suggestions😘! We are always open to contributions that improve our documentation. Please feel free to submit a pull request, and I will make it a priority to review it promptly. Looking forward to your contributions and making our docs even better together!

**Comment by AmrTamer23** [comment]
Amazing!, working on it and will submit a PR very soon.

**Comment by AmrTamer23** [comment]
I'm getting wired error like this 
![image](https://github.com/childrentime/reactuse/assets/122938074/e5b2e39d-b762-438d-b1dd-b78efd2cbbce)

and ton of warnings like this
![image](https://github.com/childrentime/reactuse/assets/122938074/67631761-1baf-4c99-897e-47e4f2c58f7c)

And it compiles with 89 errors, but that breaks the dev server after any change, tried to upgrade bunch of packages and check the files out for incorrect import statements and still the same issues, any idea how to fix them?

**Comment by childrentime** [comment]
I'm not quite sure what caused this issue. My environment is Node 20.10 on MacOS. Generally speaking

```sh
pnpm i
cd packages/website-docusaurus
pnpm dev
```

should just work.

**Comment by AmrTamer23** [comment]
Yeah, I found out it was windows only problem, tried it on WSL and it works, thanks!.
